src/library_exceptions.js

```diff
@@ -428,6 +428,17 @@ var LibraryExceptions = {
     catchInfo.free();
     {{{ makeThrow('ptr') }}}
   },
+
+  $formatException__deps: ["format_exception"],
+  $formatException: function(excPtr){
+    var stackTop = stackSave();
+    var result_ptr = stackAlloc(4);
+    {{{ exportedAsmFunc('_format_exception') }}}(result_ptr, excPtr);
```

Comment:
I think its safe to just do `_format_exception(result_ptr, excPtr)` here.  No need for the exportedAsmFunc the {{{ }}}